pkg/graveler/ref/manager.go

```diff
@@ -575,12 +568,12 @@ func (m *Manager) VerifyLinkAddress(ctx context.Context, repository *graveler.Re
 	if err != nil {
 		return err
 	}
-	return m.deleteLinkAddress(ctx, repository, token)
+	return deleteLinkAddress(ctx, m.kvStore.Store, repository, token)
 }
 
-func (m *Manager) deleteLinkAddress(ctx context.Context, repository *graveler.RepositoryRecord, token string) error {
+func deleteLinkAddress(ctx context.Context, kvStore kv.Store, repository *graveler.RepositoryRecord, token string) error {
```

Comment:
Can you please explain why you decided to modify the function to buy be a member method?

Reply:
This helper func was use from two places and I wanted to control which store is used for each case.
Called by 'VerifyLinkAddress' which is part of the public interface we should use the kv store.
Called by 'DeleteExpiredLinkAddresses' which is part of the background cleanup wanted to use the limited kv store.